BUILD

```diff
@@ -0,0 +1,13 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Is this file really required in this repository?

Reply:
I guess it isn't "required" per se. As I explained in the PR synopsis and other comments, the Bazel build is fairly useless in/of itself because there is nothing to "compile" for only headers. Therefore this PR tests that the library can be used as a dependency in a Bazel project. Without this `BUILD` file, the compilation will fail. If you're not a fan of that, then this PR can be closed since it's the crux of it.

Reply:
See https://github.com/nlohmann/json/pull/1690#discussion_r310347274